examples/00-basic/00-basic_example.py

```diff
@@ -3,14 +3,14 @@
 
 Basic DPF-Core Usage
 ~~~~~~~~~~~~~~~~~~~~
-This example shows how to open an example result file and do some
-basic post-processing.
+This example shows how to open a result file and do some
+basic postprocessing.
```

Comment:
```suggestion
basic post-processing.
```

Reply:
Replace all other occurrences of this.

Reply:
@akaszynski Is there a reason for this. The Ansys Style Guideline says to use preprocessing and postprocessing. I can send you a link to the sharepoint site with all the guides if you'd like. However, I won't fight you on this if you really want it as two words as very few writers refer to these guidelines anyhow! Also, the check “Test Core API” is failing for both Windows and Linux failed and the error messages don't mean much to me. Sorry!

Reply:
No reason other than that's how it's written in pymapdl.  We'll have to fix it there to be in compliance with the Ansys Style Guideline.

Thanks for letting me know!

Reply:
> Also, the check “Test Core API” is failing for both Windows and Linux failed and the error messages don't mean much to me. Sorry!

That's related to a change with the latest release of pyvista.  I've capped the upper limit for pyvista for now; we'll fix that in a later release.

Reply:
@akaszynski Can this PR be approved even though "Test Core API" is failing?

Reply:
> @akaszynski Can this PR be approved even though "Test Core API" is failing?

Meant to work on it this weekend.  I'd rather it pass so our merge commit history is clean.  Let me tweak something.

Reply:
Actually, CI is passing.